ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dhcp-servers.j2

```diff
@@ -0,0 +1,56 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - dhcp servers #}
+{% if dhcp_servers is arista.avd.defined %}
+{%     for dhcp_server in dhcp_servers | arista.avd.natural_sort %}
```

Comment:
This would not match EOS config order, which looks like it puts all dhcp servers with VRF before dhcp servers in default VRF. Not sure how to effectively match this order here...

Reply:
I don't quite know how to implement what you suggest, and it would not match EOS config order.

Reply:
```
host1(config-dhcp-server-vrf-defauls)#show run | se dhcp
dhcp server vrf defauls
dhcp server
dhcp server vrf defaulu
```

Reply:
That's the sorting order of EOS - if you use vrf "default", it is sorted between `defauls` and `defaulu`

Reply:
Fixed this!